non-wheel/httpcrtsrv/httpsrv.py

```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/env python3
+import optparse

```

Comment:
`optparse` is deprecated, better to use [argparse](https://docs.python.org/3.6/library/argparse.html#module-argparse) instead